Mathlib/Algebra/Ring/Semireal/Defs.lean

```diff
@@ -27,17 +27,21 @@ semireal.
 [lam_1984](https://doi.org/10.1216/RMJ-1984-14-4-767)
 -/
 
+variable {R : Type*}
+
+variable (R) in
```

Comment:
You can just have `R` explicit from the beginning (in both declarations).

Reply:
OK! And then how do I make it implicit in certain declarations?

Reply:
In the `SumsOfSquares` file, it makes more sense to me to start with `R` implicit. Is that a problem?

Reply:
Variables in different files are completely independent, so you can do whatever you want. I don't understand why you want `R` implicit in the instance.

Reply:
My bad, I was just testing out how to make implicit a variable that had been globally declared as explicit and I forgot to remove it 😖